great_expectations/core/usage_statistics/package_dependencies.py

```diff
@@ -0,0 +1,229 @@
+"""Provide GE package dependencies.
+
+This module contains static lists of GE dependencies, along with a utility for
+checking and updating these static lists.
+
+    Typical usage example:
+        ge_dependencies: GEDependencies = GEDependencies()
+        print(ge_dependencies.get_required_dependency_names())
+        print(ge_dependencies.get_dev_dependency_names())
+
+    To verify lists are accurate, you can run this file or execute main() from
+    within a cloned GE repository. This will check the existing requirements
+    files against the static lists returned via the methods above in the
+    usage example and raise exceptions if there are discrepancies.
+"""
+import os
+import re
+from typing import List, Set
+
+
+class GEDependencies:
+    """Store and provide dependencies when requested.
+
+    Also acts as a utility to check stored dependencies match our
+    library requirements.
+
+    Attributes: None
+    """
+
+    """This list should be kept in sync with our requirements.txt file."""
```

Comment:
Is there any way we can enforce this? Maybe parsing these deps and the deps in the requirements and making sure all the items noted herein are reflected in the files?

Reply:
Yep! These are enforced in CI through the `test_required_dependency_names_match_requirements_file` and `test_dev_dependency_names_match_requirements_file` integration tests. If there is a mismatch between these lists and the requirements files at test time then we fail the test. The utility in the package_dependencies file is to help identify missing packages, we can improve it to highlight differences if it becomes burdensome.